test/schema.spec.js

```diff
@@ -272,6 +275,16 @@ describe('Schema Tests', function() {
 				});
 			});
 
+			it('set the defaults for properties of type array', function(done) {
+				this.schema.create({
+					ofObjects: [{fieldA: '1234'}] 
+				}).then(function(instance) {
+					instance.should.have.deep.property('ofObjects[0].fieldA', '1234');
+					instance.should.have.deep.property('ofObjects[0].fieldB', 100);
+					done();
+				}).catch(done);
```

Comment:
No that;s the point. If we get to the point where we get to the ```catch``` clause then done will be called with one argument (the ```err```) which will make the test NOT fail but throw an error.